data_saver/tests/test_api.py

```diff
@@ -0,0 +1,21 @@
+"""
+Tests for api.py
+"""
+
+from unittest import TestCase, mock
+from data_saver import api
+
+
+class ApiTestCase(TestCase):
+
+    @mock.patch('data_saver.api.get_batch_sales_data')
+    def test_api(self, get_batch_sales_data: mock.MagicMock):
+        """
+        Test with mocked service call
+        """
+        get_batch_sales_data.return_value = [{"client": "Bob Marley", "Profit": 12}]
+        respond = api.get_sales("http://test_url", "test_token", "2022-01-01")
+
+        self.assertEqual(type(respond), type([]))
```

Comment:
Some notes:
1. `type([])` is `list`
2. Many people test return list length as well. 
3. If you need to make `get_batch_sales_data` to return different results when you call it several times in your test, you can use the [side effect](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.side_effect)
4. Also, you can use `self.assertListEquel` to test a list